drivers/storage/portworx/util/util.go

```diff
@@ -303,6 +305,39 @@ func GetPortworxVersion(cluster *corev1alpha1.StorageCluster) *version.Version {
 	return pxVersion
 }
 
+// GetStorkVersion returns the stork version based on the image provided.
+// We first look at spec.Image. If that is not present or invalid semvar, then we fallback to an
+// annotation portworx.io/stork-version; else we return int max as the version.
+func GetStorkVersion(cluster *corev1alpha1.StorageCluster) *version.Version {
+	defaultVersion, _ := version.NewVersion(strconv.FormatInt(math.MaxInt64, 10))
+	if cluster.Spec.Stork == nil && cluster.Annotations[AnnotationStorkVersion] == "" {
```

Comment:
I think if stork is disabled, we can just return the default, ignoring the annotation, right?
We should check both stork == nil and stork.enabled == false to check stork is disabled.